index.js

```diff
@@ -19,22 +19,23 @@ express()
   .set('view engine', 'ejs')
   .get('/', jsonParser, async (req, res) => {
     try {
-      const client = await pool.connect();
-      const result = await client.query(`SELECT prod_name, type, exp_dt, qty FROM fridge_products NATURAL JOIN products`);
-      const results = { 'results': (result) ? result.rows : null};
-      res.render('pages/index', results );
-      client.release();
+        const client = await pool.connect();
+        const result = await client.query(`SELECT prod_name, type, exp_dt, qty FROM fridge_products NATURAL JOIN products`);
+        const searchresult = await client.query(`SELECT * FROM products where upper(prod_name)=upper('${req.query.searchParam}')`);
```

Comment:
Add additional where condition
WHERE user_ID IN ('0', '1')

Reply:
Also, change the natural join to a left join on prod_ID

Reply:
Perfect thanks Nicole just fixed it!